README.md

```diff
@@ -245,35 +245,35 @@ const { data: app } = await requestWithAuth(
   </tr>
   <tr>
     <th align=left>
-      <code>options.mediaType.format</code>
+      <code>options.method</code>
     </th>
     <td>
       String
     </td>
     <td>
-      Media type param, such as `raw`, `html`, or `full`. See <a href="https://developer.github.com/v3/media/">Media Types</a>.
+      Any supported <a href="https://developer.github.com/v3/#http-verbs">http verb</a>, case insensitive. <em>Defaults to <code>Get</code></em>.
     </td>
   </tr>
   <tr>
     <th align=left>
-      <code>options.mediaType.previews</code>
+      <code>options.mediaType.format</code>
     </th>
     <td>
-      Array of strings
+      String
     </td>
     <td>
-      Name of previews, such as `mercy`, `symmetra`, or `scarlet-witch`. See <a href="https://developer.github.com/v3/previews/">API Previews</a>.
+      Media type param, such as `raw`, `html`, or `full`. See <a href="https://developer.github.com/v3/media/">Media Types</a>.
     </td>
   </tr>
   <tr>
     <th align=left>
-      <code>options.method</code>
+      <code>options.mediaType.previews</code>
     </th>
     <td>
-      String
+      Array of strings
     </td>
     <td>
-      Any supported <a href="https://developer.github.com/v3/#http-verbs">http verb</a>, case insensitive. <em>Defaults to <code>Get</code></em>.
+      Name of previews, such as `mercy`, `symmetra`, or `scarlet-witch`. See <a href="https://developer.github.com/v3/previews/">API Previews</a>.
```

Comment:
The docs should point that this option is for the GraphQL API only, and doesn't have an effect on REST routes

Reply:
I've updated the doc here to reflect that! Thanks.